### PR TITLE
Create yaml file for ELUC use case CI

### PR DESCRIPTION
**Pull request by danyoungday**
First try at CI.
1. Sets CWD to ELUC use case folder
2. Sets PYTHONPATH to CWD
3. Installs requirements.txt
4. Runs unittest
5. Runs pylint

### CODE REVIEW

File: .github/workflows/eluc.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    - name: Test PYTHONPATH
+      run: printenv PYTHONPATH
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install pylint
+        if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
+    - name: Run unit tests
+      run: python -m unittest
+    - name: Lint with PyLint
```
Comment (ofrancon) [inline .github/workflows/eluc.yml:34]: You should run `pylint` before the unit tests. First check the code is 'clean' (fast), then run the tests (slow).

File: .github/workflows/eluc.yml [hunk 2]
```diff
@@ -0,0 +1,35 @@
+# This runs the unit tests for the ELUC use case
+
+name: ELUC Use Case Python Unit Tests
+
+on:
+  push:
+    branches: [ "main" ]
```
Comment (ofrancon) [inline .github/workflows/eluc.yml:7]: Can we run it on all branch? Because we shouldn't be able to merge if pylint or the tests fail

**Review by ofrancon — APPROVED** [review]
looks like a good start

File: .github/workflows/eluc.yml (hunk 2 above)
Comment (danyoungday) [reply .github/workflows/eluc.yml:7]: Currently runs whenever a PR goes into main

File: .github/workflows/eluc.yml (hunk 1 above)
Comment (danyoungday) [reply .github/workflows/eluc.yml]: Moved

**Pull request merged**